### PR TITLE
added new node type (empty node)

### PR DESCRIPTION
**Pull request by nickolanack**
added a new ar method 'adNode', that creates an empty node. It can be used to group other items and is useful for creating local animations etc

### CODE REVIEW

**Comment by nickolanack** [comment]
#41 #47

File: src/ar.android.ts [hunk 1]
```diff
@@ -15,6 +17,16 @@ declare const com, android, global, java: any;
 let _fragment;
 let _origin;
 
+const addNode = (options: ARAddOptions, parentNode: SCNNode): Promise<ARGroup> => {
```
Comment (EddyVerbruggen) [inline src/ar.android.ts]: `SCNNode` is an iOS class, I think this is a copy-paste error 😉

Comment (nickolanack) [reply src/ar.android.ts]: yup. nice catch

**Comment by EddyVerbruggen** [comment]
Thanks, great idea as well!

**Pull request merged**